src/components/svg/keys/svg-keyboard-key/svg-keyboard-key.component.ts

```diff
@@ -70,6 +70,7 @@ export class SvgKeyboardKeyComponent implements OnInit, OnChanges, OnDestroy {
     @Input() width: number;
     @Input() keyAction: KeyAction;
     @Input() keybindAnimationEnabled: boolean;
+    @Input() popoverEnabled: boolean;
```

Comment:
We are passing data for`popoverEnabled` trough the inputs to the key and based on this value we decide if we would like to enable capture or not. I think that we should keep original naming of the variable, because we don't send data for capturing, but for popover and then at the end we decide based on popover functionality if we would also like to enable capturing or not.